modules/convert/user.go

```diff
@@ -13,6 +13,9 @@ import (
 // ToUser convert models.User to api.User
 // signed shall only be set if requester is logged in. authed shall only be set if user is site admin or user himself
 func ToUser(user *models.User, signed, authed bool) *api.User {
```

Comment:
```suggestion
func ToUser(doer *models.User, signed, authed bool) *api.User {
```

Reply:
@a1012112796 it converts user to apiIser we can add doer instead of signed and authed ...